index.html

```diff
@@ -3678,6 +3685,90 @@ <h3>Data Validation</h3>
 							</details>
 						</li>
 
+						<li id="validate-structural-relations">
+							<p>(<a href="#structural-rel"></a>) Verify the use of structural relations as follows:</p>
+							<ol>
+								<li id="rel-booleans">
+									<p>Let <var>contents</var>, <var>pagelist</var> and <var>cover</var> be <a
+											href="https://infra.spec.whatwg.org/#boolean">boolean</a> values set to
+										false.</p>
+								</li>
+								<li id="rel-create-list">
+									<p>Let <var>resources</var> be the result of <a
+											href="https://infra.spec.whatwg.org/#list-extend">extending</a>
+										<var>data["readingOrder"]</var> with <var>var["resources"]</var>.</p>
+								</li>
```

Comment:
The [infra "extending"](https://infra.spec.whatwg.org/#list-extend) operation says:

> To extend a list A with a list B, for each item of B, append item to A.

in human readable:-) term, this means `A = A append B`. 

In our case that can be misrepresented, and a false reading would yield an extended `data["readingOrder"]`. Clearly, any mindful implementer will realize that we do a run through the list without that assignment, we may want to be a bit more careful in the formulation.

Reply:
Sure, we could assign readingorder to resources first and then extend.

Reply:
Actually, this makes sense as the reading order and resource list might not be set, so it allows some default checking on both:

> Set `resources` to the value of `data["readingOrder"]`, when defined, otherwise to an empty ordered set. Extend `resources` with `var["resources"]`, when defined.

Reply:
Which is exactly how I implemented it:-) 👍